recipes/arrow-adbc/meta.yaml

```diff
@@ -0,0 +1,203 @@
+      run:
+    test:
+      commands:
+        - test -f $PREFIX/include/adbc.h                                         # [unix]
+        - test -f $PREFIX/include/adbc_driver_manager.h                          # [unix]
+        - test -d $PREFIX/lib/cmake/AdbcDriverManager/                           # [unix]
+        - test -f $PREFIX/lib/pkgconfig/adbc-driver-manager.pc                   # [unix]
+        - test ! -f $PREFIX/lib/libadbc_driver_manager.a                         # [unix]
+        - test -f $PREFIX/lib/libadbc_driver_manager.so                          # [linux]
+        - test -f $PREFIX/lib/libadbc_driver_manager.dylib                       # [osx]
+
+        - if not exist %LIBRARY_BIN%\adbc_driver_manager.dll exit 1              # [win]
+        - if not exist %LIBRARY_INC%\adbc.h exit 1                               # [win]
+        - if not exist %LIBRARY_INC%\adbc_driver_manager.h exit 1                # [win]
+        - if not exist %LIBRARY_LIB%\adbc_driver_manager.lib exit 1              # [win]
+        - if not exist %LIBRARY_LIB%\cmake\AdbcDriverManager exit 1              # [win]
+        - if not exist %LIBRARY_LIB%\pkgconfig\adbc-driver-manager.pc exit 1     # [win]
+
+  - name: adbc-driver-postgresql-cpp
+    script: build-cpp.sh                                                         # [not win]
+    script: build-cpp.bat                                                        # [win]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - ninja
+        - pkg-config
+      host:
+        - libpq
+      run:
+        - libpq
+    test:
+      commands:
+        - test ! -f $PREFIX/include/adbc.h                                       # [unix]
+        - test -d $PREFIX/lib/cmake/AdbcDriverPostgreSQL/                        # [unix]
+        - test -f $PREFIX/lib/pkgconfig/adbc-driver-postgresql.pc                # [unix]
+        - test ! -f $PREFIX/lib/libadbc_driver_postgresql.a                      # [unix]
+        - test -f $PREFIX/lib/libadbc_driver_postgresql.so                       # [linux]
+        - test -f $PREFIX/lib/libadbc_driver_postgresql.dylib                    # [osx]
+
+        - if exist %LIBRARY_INC%\adbc.h exit 1                                   # [win]
+        - if not exist %LIBRARY_BIN%\adbc_driver_postgresql.dll exit 1           # [win]
+        - if not exist %LIBRARY_LIB%\adbc_driver_postgresql.lib exit 1           # [win]
+        - if not exist %LIBRARY_LIB%\cmake\AdbcDriverPostgreSQL exit 1           # [win]
+        - if not exist %LIBRARY_LIB%\pkgconfig\adbc-driver-postgresql.pc exit 1  # [win]
+
+  - name: adbc-driver-sqlite-cpp
+    script: build-cpp.sh                                                         # [not win]
+    script: build-cpp.bat                                                        # [win]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
```

Comment:
There are linker warnings that `libstdcxx` is unused by the sqlite library. Either remove this compiler or add `ignore_run_exports_from: {{ compiler('cxx') }}`.